### PR TITLE
Fix for issue #685 OpenRoberta/openroberta-lab shortcut messages added

### PR DESCRIPTION
**Pull request by Prantikc22**
all messages regarding the shortcut feature has been added.

### CODE REVIEW

File: robMsg/json/de.json [hunk 1]
```diff
@@ -134,6 +134,17 @@
 	"LOOP_FOREVER": "Wiederhole unendlich oft",
 	"MENU_ABOUT": "Über das Open Roberta Lab",
 	"MENU_ABOUT_PROJECT": "Über das Open Roberta Projekt",
+	"MENU_SHORTCUT_IMPORT": "Ctrl + I: Programm importieren",
```
Comment (bjost2s) [inline robMsg/json/de.json:137]: Please use here "Importiere Programm" as it is defined in the German translation of the MENU_IMPORT_PROG key

File: robMsg/json/de.json [hunk 2]
```diff
@@ -134,6 +134,17 @@
...
+	"MENU_SHORTCUT_EXPORT": "Ctrl + E: Programm exportieren",
```
Comment (bjost2s) [inline robMsg/json/de.json:138]: Please use here "Exportiere Programm" as it is defined in the German translation of the MENU_EXPORT_PROG key

File: robMsg/json/de.json [hunk 3]
```diff
@@ -134,6 +134,17 @@
...
+	"MENU_SHORTCUT_SAVE": "Ctrl + S: Speichern",
+	"MENU_SHORTCUT_SAVEAS": "Ctrl + Shift + S: Speichern als",
```
Comment (bjost2s) [inline robMsg/json/de.json:140]: Please use here "Speichern unter" as it is defined in the German translation of the MENU_SAVE_AS key

File: robMsg/json/de.json [hunk 4]
```diff
@@ -134,6 +134,17 @@
...
+	"MENU_SHORTCUT_RUN": "Ctrl + R: Auf Roboter laufen",
```
Comment (bjost2s) [inline robMsg/json/de.json:141]: Please use here "Starte auf dem Roboter"

File: robMsg/json/de.json [hunk 5]
```diff
@@ -134,6 +134,17 @@
...
+	"MENU_SHORTCUT_SOURCE_CODE": "Ctrl + Z: Zeige Quellcode",
+	"MENU_SHORTCUT_EDITOR": "Ctrl + Shift + Z: Quellcode-Editor",
```
Comment (bjost2s) [inline robMsg/json/de.json:143]: Please use here "Öffne QuellcodeEditor" as it is defined in the German translation of the MENU_SOURCE_CODE_EDITOR key

File: robMsg/json/de.json [hunk 6]
```diff
@@ -134,6 +134,17 @@
...
+	"MENU_SHORTCUT_MULTISIM": "Ctrl + Shift + G: Mehrfachrobotersimulation",
```
Comment (bjost2s) [inline robMsg/json/de.json:144]: Please use here "Mehrere Roboter simulieren" as it is defined in the German translation of the MENU_RUN_MULT_SIM key

File: robMsg/json/de.json [hunk 7]
```diff
@@ -134,6 +134,17 @@
...
+	"MENU_SHORTCUT_OPEN_SIM": "Ctrl + G: Simulationsansicht",
```
Comment (bjost2s) [inline robMsg/json/de.json:145]: Please use here "Öffne/schließe die Simulation" as it is defined in the German translation of the MENU_START_SIM key

File: robMsg/json/de.json [hunk 8]
```diff
@@ -134,6 +134,17 @@
...
+	"MENU_SHORTCUT_VIEW_PROGRAM": "Ctrl + M: Meine Programme",
+	"MENU_SHORTCUT": "Tastatürkürzel",
```
Comment (bjost2s) [inline robMsg/json/de.json:147]: Please correct the spelling "Tastaturkürzel"

**Comment by Prantikc22** [comment]
Spellings has been corrected now.

**Pull request merged**